posthog/constants.py

```diff
@@ -15,7 +15,8 @@ class AvailableFeature(str, Enum):
     PATHS_ADVANCED = "paths_advanced"
     CORRELATION_ANALYSIS = "correlation_analysis"
     GROUP_ANALYTICS = "group_analytics"
-    MULTIVARIATE_FLAGS = "multivariate_flags"
+    MULTIVARIATE_FLAGS = ("multivariate_flags",)
```

Comment:
I'm guessing you didn't actually want to add a comma (which I expect got turned into this via lint)

Reply:
good catch, fixed